src/model/user/profile.rs

```diff
@@ -1,9 +1,6 @@
 use crate::model::user::{Color, ModLevel};
-use serde::{Deserialize, Serialize};
-use std::{
-    borrow::Cow,
-    fmt::{Display, Formatter},
-};
+use serde::{__private::Formatter, Deserialize, Serialize};
```

Comment:
wrong import?

Reply:
Which import do we not need here? is it (De)Serialize?

Reply:
If this is about the __private::Formatter, serde complains when __private is not specified.

Reply:
it should be `use std::fmt::Formatter` if I'm not mistaken, the `serde::__private::Formatter` is just a re-export of the standard library. For some reason IntelliJ prefers to import that one, no idea why.